airbyte-integrations/connectors/source-google-drive/source_google_drive/spec.py

```diff
@@ -67,22 +67,19 @@ class Config:
     def documentation_url(cls) -> str:
         return "https://docs.airbyte.com/integrations/sources/google-drive"
 
-    @staticmethod
```

Comment:
Will this have to be reverted when the platform issue is fixed? Couldn't quite see why google drive had this method but the other sources don't.

Reply:
This has been moved into the abstract file based spec. It's only on google drive because the other connectors didn't have oneOfs in their spec which made it necessary (Google drive has two auth modes)